### PR TITLE
Chore/18/feed layout

### PR DESCRIPTION
**Pull request by ethanswift**
**What this PR does**:
Views which make up a Post, such as avatar, avatar view, type, title, body and ...
**Which issue(s) this PR fixes**:
part of the layout for the feed 
Fixes #

**Special notes for reviewers**:
the feed view controller is just for testing these views. 
post container view is a container view for the whole components of the post
the footer post view is incomplete 
minor issues with fonts hasn't yet been resolved
auto layout hasn't been tested on more variety of situations
overall it's not that clean yet!
**Additional comments**:

### CODE REVIEW

**Comment by haidalinda** [comment]
@ethanswift before submitting a PR please:
* Address any Xcode warnings you introduced
* Address any SwiftLint warnings you introduced
* Remove commented code
* Remove print statements

File: FightPandemics/Post/FeedPostFooter.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+//        sendImgView.frame.size.width = 20
+//        sendImgView.frame.size.height = 17
+//        let sendLbl = UILabel()
+//        sendLbl.text = "\(self.sendNum)"
+//        sendLbl.font = UIFont(name: "System", size: 14)
+//        sendLbl.textAlignment = .center
+//        sendLbl.textColor = UIColor(hexString: "#939393")
+//        let firstStackView = UIStackView()
+    }
+}
```
Comment (haidalinda) [inline FightPandemics/Post/FeedPostFooter.swift]: it looks like `FeedPostFooter` isn't used - i'd suggest removing the `FeedPostFooter.swift` file

File: FightPandemics/Post/FeedPost.swift [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                moreButton.translatesAutoresizingMaskIntoConstraints = false
+        //        NSLayoutConstraint(item: moreButton, attribute: .width, relatedBy: .equal, toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: 100).isActive = true
+        //        NSLayoutConstraint(item: moreButton, attribute: .height, relatedBy: .equal, toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: 22).isActive = true
+                NSLayoutConstraint(item: moreButton, attribute: .left, relatedBy: .equal, toItem: self, attribute: .left, multiplier: 1, constant: 24).isActive = true
+                NSLayoutConstraint(item: moreButton, attribute: .right, relatedBy: .equal, toItem: self, attribute: .right, multiplier: 1, constant: 24).isActive = true
+                NSLayoutConstraint(item: moreButton, attribute: .bottom, relatedBy: .equal, toItem: self, attribute: .bottom, multiplier: 1, constant: 0).isActive = true
+        //        NSLayoutConstraint(item: moreButton, attribute: .bottom, relatedBy: .equal, toItem: self, attribute: .bottom, multiplier: 1, constant: 0).isActive = true
```
Comment (haidalinda) [inline FightPandemics/Post/FeedPost.swift]: remove commented lines

File: FightPandemics/Post/FeedPost.swift [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        super.init(frame: CGRect())
+        setUp()
+    }
+    func setUp () {
+//        self.frame = CGRect.zero
+//        self.frame.origin.x = 23
+//        self.frame.origin.y = 23
+//        self.frame.size.width = CGFloat(UIScreen.main.bounds.width - 47)
+        // it has a fixed height which I don't know ; or it shoudl be calculated the sme way
+//        self.frame.size.height = 300
```
Comment (haidalinda) [inline FightPandemics/Post/FeedPost.swift]: remove commented lines

File: FightPandemics/Post/PostContainer.swift [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+            self.frame.origin.x = (self.rect?.origin.x)!
+            self.frame.origin.y = (self.rect?.origin.y)!
+            self.frame.size.width = (self.rect?.size.width)!
+            self.frame.size.height = (self.rect?.size.height)!
+        } else {
+            self.frame.origin.x = 0
+            self.frame.origin.y = 0
+            self.frame.size.width = UIScreen.main.bounds.width
+            self.frame.size.height = height
+        print(self.frame)
```
Comment (haidalinda) [inline FightPandemics/Post/PostContainer.swift]: remove print statement

File: FightPandemics/Post/FeedViewController.swift [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+    /*
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Get the new view controller using segue.destination.
+        // Pass the selected object to the new view controller.
+    }
+    */
```
Comment (haidalinda) [inline FightPandemics/Post/FeedViewController.swift]: remove this commented code

File: FightPandemics/Post/FeedViewController.swift [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        self.postContainer.frame.origin.y = 100
+        self.view.addSubview(postContainer)
+        self.feedPost = FeedPost(type: "  Medical Supplies .", title: title, body: body)
+        postContainer.addSubview(avatar)
+        postContainer.addSubview(feedPost)
+        // Do any additional setup after loading the view.
+}
+    override func viewDidLayoutSubviews() {
+//        avatar.draw(CGRect.zero)
+    }
```
Comment (haidalinda) [inline FightPandemics/Post/FeedViewController.swift]: remove the `viewDidLayoutSubviews()` method

**Review by haidalinda — CHANGES_REQUESTED** [review]

File: FightPandemics/Post/FeedPostFooter.swift (hunk 1 above)
Comment (ethanswift) [reply FightPandemics/Post/FeedPostFooter.swift]: it's incomplete, I'll remove it

**Comment by haidalinda** [comment]
going to close per our discussion @ethanswift

**Review by stavares843 — CHANGES_REQUESTED** [review]
1 - footer on the bottom on the text

https://jmp.sh/NdgPSgh

Screenshot 2020-05-10 at 18 12 54

2 - spacing between avatar and name
3 - blue dot below name not centered with subtitle
4 - font overall not the same compared with figma
https://www.figma.com/proto/kT6SJLL4v3w06yFF2cGIun/FightPandemics-MVP?node-id=473%3A1525&scaling=min-zoom



Screenshot 2020-05-10 at 19 20 13

iPhone 11 max pro, iOS 13

**Pull request closed without merging**